src/dotnet/commands/dotnet-vstest/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright(c) .NET Foundation and contributors.All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Cli;
+
+namespace Microsoft.DotNet.Tools.VSTest
+{
+    public class VSTestCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication cmd = new CommandLineApplication(throwOnUnexpectedArg: false)
+            {
+                Name = "vstest",
+                FullName = "vstest.console",
+                Description = "Test Driver for the .NET Platform"
+            };
+
+            cmd.HelpOption("-h|--help");
+
+            var arguments = cmd.Argument(
```

Comment:
Make sense @eerhardt . We have removed this argument. Now we parsing only help option(-h|--help) and calling vstest.console.dll with the arguments which user is providing. 
Fixed in next commit.